### DISCUSSION TITLE
3.0.0 release

### DISCUSSION THREAD

**Discussion by KoditkarVedant** [notion-dotnet/notion-sdk-net#277] [Announcements]
### Enhancements ✨ 
- #241 Support Notion-Version 2022-02-22
  - #242 The `text` in blocks has been renamed to `rich_text`, to be consistent with the database property type
  - #243 Query database filter changes
    - #246 `phone` and `text` no longer supported in the query database endpoint
    - #247 rollup property filters accept `rich_text` instead of `text`
    - #248 `formula` property filters accept `string` instead of `text`
  - #244 `property_item` objects now return a `type`, `next_url`, and `id`
  - #245 Removed the List Databases endpoint

### Bug Fixes 🐛 
- N/A

### CI/Test 👷🏼 
- N/A

### Documentations 📝 
- Added details of default Notion-Version supported by NuGet package versions.

It got delayed but It's finally here. Now the library supports Notion-Version 2022-02-22. 
Once again thank you all for the love and support you are showing on this SDK. Everyone are welcome to contribute. I'm happy to hear any idea you have - fill free to open issue or discussion.
🎉 🎉 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/notion-dotnet/notion-sdk-net/releases/tag/3.0.0'>3.0.0 release</a>.</em>

**Comment by sebArtisan** [comment]
Hi, really go work. 
Are you planning to support Notion version 2022-06-28  ?
Thanks

**Reply by KoditkarVedant** [reply]
Yes, I have created a a branch `3.1.0` as per milestone - I just need some time from my regular work to work on the issues. I am open If you inclined to raise PR for any issue you like.
